public/app/percona/inventory/Tabs/Agents.tsx

```diff
@@ -33,20 +31,20 @@ import { GET_AGENTS_CANCEL_TOKEN, GET_NODES_CANCEL_TOKEN, GET_SERVICES_CANCEL_TO
 import { Messages } from '../Inventory.messages';
 import { InventoryService } from '../Inventory.service';
 
-import { beautifyAgentType, getAgentStatusColor, toAgentModel } from './Agents.utils';
-import { formatNodeId } from './Nodes.utils';
+import { beautifyAgentType, getAgentStatusColor, getAgentStatusText, toAgentModel } from './Agents.utils';
+import { getTagsFromLabels } from './Services.utils';
 import { getStyles } from './Tabs.styles';
 
 export const Agents: FC<GrafanaRouteComponentProps<{ serviceId: string; nodeId: string }>> = ({ match }) => {
   const [agentsLoading, setLoading] = useState(false);
   const [modalVisible, setModalVisible] = useState(false);
   const [data, setData] = useState<Agent[]>([]);
   const [selected, setSelectedRows] = useState<any[]>([]);
-  const serviceId = match.params.serviceId ? formatServiceId(match.params.serviceId) : undefined;
+  const serviceId = match.params.serviceId ? match.params.serviceId : undefined;
```

Comment:
I think we don't need the variable anymore as it was used to format the service id.

Reply:
@YashSartanpara1

Reply:
I think we can keep the variable to simplify the use of serviceId in the code below? i will remove the ternary operator and direct assign the value to variable.

Reply:
I would remove it as it redundant.